platform/feature-api/api/pod-api/src/api.ts

```diff
@@ -131,22 +131,25 @@ export interface PolyOut extends Omit<FS, "readdir"> {
  */
 export interface PolyNav {
     /**
-     * A way for features to display a contents of a web page
+     * A way for features to display a contents of a web page of the given URL.
+     * @param {string} url - The URL to open.
      */
     openUrl(url: string): Promise<void>;
     /**
-     * Describe what actions are possible within the pod when a feature is loaded
+     * Describe what keyboard navigation actions are possible within the pod when a feature is loaded
+     * @param {string[]} actions - A list of actions that the user can take.
      */
     setActiveActions(actions: string[]): Promise<void>;
     /**
      * Set a title in of a Pod
+     * @param {string} title - The title to set.s
      */
     setTitle(title: string): Promise<void>;
     /**
-     * Ask the user to pick a file
-     * @param type the type of file the user is asked to select, as a valid MIME type string. If no type is passed, the user can chose any type of file.
+     * Ask the user to pick a file via a dialog and returns the selected file.
```

Comment:
It's not necessarily a dialog, that's just how it works on current polyPod implementation. But I guess we could change it to `Ask the user to pick a file and return it`, thought that seems a bit obvious to me.